Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Managers/AssetsManager.swift

```diff
@@ -23,6 +24,25 @@ extension PrimerHeadlessUniversalCheckout {
             return UIImage(named: "\(cardNetwork.rawValue)-logo-colored", in: Bundle.primerResources, compatibleWith: nil)
         }
         
+        public static func getCardNetworkAssets(for cardNetwork: CardNetwork) throws -> PrimerCardNetworkAsset? {
```

Comment:
Nit-pick: This method is named `getCardNetworkAssets` but returns a singular `PrimerNetworkAsset`, potentially misleading 🤔